spec/integration/lib/auth0/api/v2/api_resource_servers_spec.rb

```diff
@@ -98,6 +98,33 @@
     end
   end
 
+    describe '.patch_client', vcr: true do
```

Comment:
Just noticed ... indented too far

Reply:
Fixed this. Also changed the description string to say `.patch_resource_server` as it should.